x-pack/filebeat/module/aws/elb/ingest/pipeline.yml

```diff
@@ -139,6 +139,26 @@ processors:
       target_field: source.as.organization.name
       ignore_missing: true
 
+  - set:
+      field: tls.cipher
+      value: '{{aws.elb.ssl_cipher}}'
+      if: ctx.aws?.elb.ssl_cipher != null
```

Comment:
Does this work safely if the `ssl_cipher` field does not exist? 
Is this the same as `if: ctx.aws?.elb?.ssl_cipher != null`?

Reply:
It does work if `ssl_cipher` doesn't exist, the http logs are good examples.

That being said it might be safer to use null safe operator on both

This doc would fails with current `if: ctx.aws?.elb.ssl_cipher != null`
```
{
  "aws.elb.ip": "10.0.0.1"
}
```

and this doc succeeds
```
{
  "aws": {
    "elb": {
      "ip": "10.0.0.1"
    }
  }
}
```

both work with `if: ctx.aws?.elb?.ssl_cipher != null`

Reply:
> This doc would fails...

This is to be expected, Elasticsearch ingest pipelines don't support dotted keys (other than the dot expander processor). So I'd be surprised if Painless (the if clause) supported it.